iceoryx_posh/source/capro/service_description.cpp

```diff
@@ -104,32 +103,17 @@ ServiceDescription::ServiceDescription(const cxx::Serialization& f_serial) noexc
 }
 
 ServiceDescription::ServiceDescription() noexcept
-    : ServiceDescription(InvalidID, InvalidID, InvalidID)
+    : ServiceDescription(InvalidString, InvalidString, InvalidString)
 {
 }
 
-ServiceDescription::ServiceDescription(uint16_t f_serviceID, uint16_t f_instanceID) noexcept
-    : ServiceDescription(f_serviceID, InvalidID, f_instanceID)
-{
-    m_hasServiceOnlyDescription = true;
-}
-
+/// @todo remove
```

Comment:
please always add an issue number to a todo

Reply:
I agree with you. Next time I'll add those. However, in this case all `@todo`'s are already removed in either #853 or #854 so there's no need to track them. If I add them now, this will result in merge conflicts.